k-distribution/tutorial/1_k/3_lambda++/lesson_5/Makefile.det

```diff
@@ -3,6 +3,7 @@ TESTDIR=tests/deterministic
 DEFDIR=det
 KOMPILE_BACKEND=java
 include $(MAKEFILE_PATH)/../../1_lambda/lesson_1/Makefile
+KOMPILE_FLAGS=
```

Comment:
Is this needed?

Reply:
Yes; it disables the bison parser on a lesson that compiles with the Java backend and thus cannot use it.

Reply:
Can you add a simple comment to this line to state that? It's not obvious and might cause headaches in the future.
Something like:
`# using the java backend so disable bison parser`

Reply:
Seconded adding the comment